woocommerce/woocommerce-android
# [Pay in Person] Change "Pay in Person" error text and illustration

Closes: #10361 


### Description

This PR changes the illustration and the error hint to better convey that Pay in Person is not mandatory to collect IPP for orders created in-app.

Design: w3JBReW1UJesCf02sj43kR-fi-1445%3A15221



### Testing instructions

1. Disable COD option either in the web admin or from the payments hub screen
2. Navigate to the payments hub screen
3. Ensure you see a message at the bottom which indicates that the onboarding is not complete
4. Click on it
5. Ensure you see "Pay in Person" error page
6. Ensure that the illustration and the text are as per design.
7. Ensure the error screen looks fine in dark mode



## Screenshots

**Before Changes** | **After Changes**
:-----------------:|:-----------------:
![Before](https://github.com/woocommerce/woocommerce-android/assets/1331230/c1b49e1e-aa6f-4f3a-8206-22215530947d) | ![After](https://github.com/woocommerce/woocommerce-android/assets/1331230/20f3cfbc-01b2-4efa-b342-c5a2d28a52cc)
